app/code/Magento/Inventory/Model/StockSourceLink/Command/Save.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Inventory\Model\StockSourceLink\Command;
+
+use Magento\Framework\Exception\CouldNotSaveException;
+use Magento\Inventory\Model\ResourceModel\StockSourceLink as StockSourceLinkResourceModel;
+use Magento\InventoryApi\Api\Data\StockSourceLinkInterface;
+use Psr\Log\LoggerInterface;
+
+/**
+ * @inheritdoc
+ */
+class Save implements SaveInterface
```

Comment:
Pls use interfaces from
https://github.com/magento-engcom/msi/issues/347

For example, for save need to use `StockSourceLinksSaveInterface`
Need to fix in all places